Src/UIComponents/utils/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const createElement = (type: string, className: string = '') => {
```

Comment:
worth adding a return type for this function

Reply:
This file was removed from my pull request.